test/timer.c

```diff
@@ -0,0 +1,54 @@
+#include "common/common.h"
```

Comment:
There are few warnings introduced in this file, could you check CI logs and fix them?